content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,174 @@
+  arrows on either side of the month to navigate to the desired month, then click on the
+  :guilabel:`day` to select the date. If no entry is selected that indicates the applicant is ready
+  to begin work immediately.
+- :guilabel:`Tags`: select as many tags as desired from the drop-down menu. To add a tag that does
+  not exist, type in the tag name, then click :guilabel:`Create "new tag"`.
+
+Job section
+-----------
+
+The following fields are pre-populated when creating a new applicant, as long as these field are
+specified on the Job Position. Editing the fields is possible, if desired.
+
+- :guilabel:`Applied Job`: select the job position the applicant is applying to from the drop-down
+  menu.
+- :guilabel:`Department`: select the department the job position falls under from the drop-down
+  menu.
+- :guilabel:`Company`: select the company the job position is for using the drop-down menu. This
+  field only appears when in a multi-company database.
+
+Contract section
+----------------
+
+- :guilabel:`Expected Salary`: enter the amount the applicant is requesting for the role in this
+  field. The number should be in a `XX,XXX.XX` format. The currency is determined by the
+  localization setting for the company.
+- :guilabel:`Extra advantages...`: if any extra advantages are requested by the applicant, enter it
+  in the :guilabel:`Extra advantages...` field to the right of the :guilabel:`Expected Salary`
+  field. This should be short and descriptive, such as `1 week extra vacation` or `dental plan`.
+- :guilabel:`Proposed Salary`: enter the amount to be offered to the applicant for the role in this
+  field. The number should be in a `XX,XXX.XX` format. The currency is determined by the
+  localization setting for the company.
+- :guilabel:`Extra advantages...`: if any extra advantages are offered to the applicant, enter it in
+  the :guilabel:`Extra advantages...` field to the right of the :guilabel:`Proposed Salary` field.
+  This should be short and descriptive, such as `unlimited sick time` or `retirement plan`.
+
+Application summary tab
+-----------------------
+
+Any additional details or notes that should be added to the applicant's card can be typed into this
+field.
+
+Skills tab
+----------
+
+Skills can be added to the applicant's card. To add a skill, follow the same steps as outlined in
+the skills section of the :ref:`Create new employees <employees/skills>` document.
+
+.. image:: add-new-applicants/new-applicant.png
+   :align: center
+   :alt: All the fields for a new applicant form entered.
+
+.. _recruitment/quick-add-applicant:
+
+Quick add
+=========
+
+Quickly add a new applicant using the *quick add* button. If not already in the job position Kanban
+view, navigate to the main recruitment dashboard by going to :menuselection:`Recruitment app -->
+Applications --> By Job Positions`. Click the :guilabel:`(#) New Applications` button on the job
```

Comment:
To make the document skimmable, I'd start with the navigation instructions, then mention the quick add button (: